tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Http/WebRequest/WebRequestCommon.cs

```diff
@@ -55,7 +56,8 @@ public static CallTargetState GetResponse_OnMethodBegin<TTarget>(TTarget instanc
                     {
                         if (setSamplingPriority && spanContext?.SamplingPriority is not null)
                         {
-                            scope.Span.SetTraceSamplingPriority(spanContext.SamplingPriority.Value);
+                            // TODO: figure out SamplingMechanism, do we propagate that as well for special cases like this?
```

Comment:
TODO: figure out SamplingMechanism, do we propagate that as well for special cases like this?